src/store/validation/index.ts

```diff
@@ -106,31 +106,39 @@ export const actions: ActionTree<IValidationState, IRootState> = {
   },
 
   /**
-   * Validate blocks from backend SAVE action, pick info from from active flow's vendor_metadata:
+   * Validate blocks from backend SAVE action.
+   */
+  async validate_allBlocksFromBackend({dispatch}): Promise<void> {
+    await dispatch('validate_fromBackend', {type: 'block'})
+    await dispatch('validate_fromBackend', {type: 'resource'})
+  },
+
+  /**
+   * Validate block/resource form backend, pick info from active flow's vendor_metadata:
    * floip: {
    *   validation_results: {
    *      blocks: {
-   *         [`${block.uuid}`]: [
-   *             { message: 'validation error #1 from backend' },
-   *             { message: 'validation error #2 from backend' },
-   *         ]
+   *         [`${block.uuid}`]: [{ message: 'validation error #1 from backend' },{ message: 'validation error #2 from backend' },]
+   *     }
+   *     resources: {
+   *         [`${resource.uuid}`]: [{ message: 'validation error #1 from backend' },{ message: 'validation error #2 from backend' },]
    *     }
    *   }
    * }
    */
-  async validate_allBlocksFromBackend({state, rootGetters}): Promise<void> {
-    const backendErrorsList = get(rootGetters['flow/activeFlow']?.vendor_metadata, 'floip.ui_metadata.validation_results.blocks', [])
+  async validate_fromBackend({state, rootGetters}, {type}: {type: 'block' | 'resource'}): Promise<void> {
+    const backendErrorsList = get(rootGetters['flow/activeFlow']?.vendor_metadata, `floip.ui_metadata.validation_results.${type}s`, [])
```

Comment:
It would be nice to have at least _some_ types here, and probably change the default value to `{}`, like
```
  const backendErrorsList = get(
      rootGetters['flow/activeFlow'],
      `vendor_metadata.floip.ui_metadata.validation_results.${type}s`,
      {},
  ) as Record<string, {message: string}[]>
```

Reply:
Thanks @bulatgab , the default value is definitely `{}`
And good suggestion for the typing.

I'm making a change. 💯